h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/XGBoost.java

```diff
@@ -252,6 +252,14 @@
     PlattScalingHelper.initCalibration(this, _parms, expensive);
   }
 
+  static boolean allowMultiGPU() {
+    return H2O.getSysBoolProperty("xgboost.multinode.gpu.enabled", false);
+  }
+
+  private static boolean gpuCheckEnabled() {
```

Comment:
nit: Is the GPU check related to multinode or is it generic. The name of the method sounds generic, but the property name is related to multinode,

Reply:
Yes, good idea - I need to fix the property name